libsemigroups_pybind11/fpsemigroup.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, M. T. Whyte
+#
+# Distributed under the terms of the GPL license version 3.
+#
+# The full license is in the file LICENSE, distributed with this software.
+
+# pylint: disable=no-name-in-module, invalid-name, unused-import, fixme
+
+"""
+This package provides the user-facing python part of libsemigroups_pybind11 for
+various examples of presentations for finitely presented semigroups and monoids.
+"""
+
+from _libsemigroups_pybind11 import (
+    make,
```

Comment:
Is this required? What is it? Seems to me overwritten in line 41 below, but I might be wrong.